api/src/test/java/io/strimzi/test/k8s/BaseKubeClient.java

```diff
@@ -203,12 +203,17 @@ public K deleteNamespace(String name) {
     }
 
     @Override
-    public ProcessResult exec(String pod, String... command) {
+    public ProcessResult execInPod(String pod, String... command) {
         List<String> cmd = namespacedCommand("exec", pod, "--");
         cmd.addAll(asList(command));
         return Exec.exec(cmd);
     }
 
+    @Override
+    public ProcessResult exec(String... command) {
```

Comment:
TBH, I'm not sure whether this belongs into `BaseKubeCLient` since it is just local exec and has nothing to do with Kubernetes client, or? But I can live with it if it works for others.

Reply:
@scholzj Good comment, I thought about it but I had a dilemma: make class `Exec` as `public`(and general method to execute commands) or have general `exec` method in class `BaseKubeClient`that can mean that I want to execute some command on the VM with kubernetes

Reply:
Yeah ... as I said ... I'm fine with this ... not forcing you to change it. It was just an observation.